mitype/commandline.py

```diff
@@ -155,3 +170,30 @@ def load_based_on_difficulty(difficulty_level=random.randrange(1, 6)):
 
     print("Select a difficulty level in range [1,5]")
     sys.exit(2)
+
+
+def show_history(N):
+    N = abs(int(N))
```

Comment:
Do not use abs here, as we are using -1 as default value for N ad then detecting this value at line:
```
start_count = 0 if N >= len(data) or N == -1 else no_of_records-N
```
If user enters negative number, just quit the program saying:
`Please enter a positive integer`

Now there's a possibility user enters -1, but even in that case we still print all the history.

Reply:
Ohh yes, my bad. I'll fix it.